### DISCUSSION TITLE
Who is this İbax?

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#1083] [Q&A] [answered]
Everyone is talking about how good an Ibax project is. As much as I want to believe this, there are some situations that catch my eye. I want to list them..

1-)Who is behind İbax? Who are its supporters?

2-) İbax telegram or Discord Admins are not active enough When a question is asked, there is no answer and we are at the beginning of the project

These two questions are my basic thoughts, if you have any information, please contact...
-Thank you for reading. Goodbye.

![wallpaper-3584230_1920](https://user-images.githubusercontent.com/98268472/151365703-436e90ad-2887-42b5-84e1-5d7c913dd2ac.jpg)

**Comment by ghost** [comment, marked as answer]
You're right about that, my friend, we don't know much.

**Comment by yasirozturk01** [comment]
All will be revealed soon
